packages/app/src/App.tsx

```diff
@@ -99,6 +102,9 @@ const AppRouter = app.getRouter();
 const routes = (
   <FlatRoutes>
     <Navigate key="/" to="/catalog" />
+    <Route path="/home" element={<HomepageCompositionRoot />}>
```

Comment:
From earlier comments in this PR: it is a bit odd for it not to be there, but Tim's comments about evaluators made sense too.  Maybe just a `todo` to indicate the homepage will be moved to the actual root when it is more mature?

Reply:
Yes, good call. Fixed in f828b35.